lib/senders/index.js

```diff
@@ -46,15 +46,6 @@ module.exports = function (log, config, error, bounces, translator, sender) {
       }, [])
     }
 
-    function getSecondaryEmails(emails) {
```

Comment:
Tangential but while you're in here removing this, maybe `getVerifiedSecondaryEmails` could be simplified to use `Array.filter` instead of `Array.reduce`?